foundryup/foundryup

```diff
@@ -23,6 +24,8 @@ main() {
       -r|--repo)        shift; FOUNDRYUP_REPO=$1;;
       -b|--branch)      shift; FOUNDRYUP_BRANCH=$1;;
       -v|--version)     shift; FOUNDRYUP_VERSION=$1;;
```

Comment:
should we rename `--version` to `--install`? considering we are already making a breaking change

Reply:
good point, done in https://github.com/foundry-rs/foundry/pull/9551/commits/bf24bd9155a42c152d290771f0a0496ada8ee46a